tools/table_annovar/table_annovar.xml

```diff
@@ -42,7 +42,7 @@
         #set operation = ','.join( $operation )
 
         ## Annotate variants.
-        table_annovar.pl ${tab_anno_input} ${__get_data_table_entry__('annovar_indexes', 'dbkey', $input.dbkey, 'path')} -protocol ${protocol} -operation ${operation} -nastring '.' -buildver ${input.dbkey} --outfile output
+        \${SCRIPT_PATH}/table_annovar.pl ${tab_anno_input} ${__get_data_table_entry__('annovar_indexes', 'dbkey', $input.dbkey, 'path')} -protocol ${protocol} -operation ${operation} -nastring '.' -buildver ${input.dbkey} --outfile output
```

Comment:
With this `\${SCRIPT_PATH}/ ` in place, users can't append the scripts to the path anymore, since when  ${SCRIPT_PATH} is not set, it will read `/table_annovar.pl`. You could change this to \${SCRIPT_PATH}table_annovar.pl and document that SCRIPT_PATH needs to include the trailing slash. But then again I'm not quite sure why the script path can't be appended to PATH ?!